src/feedback.rs

```diff
@@ -9,23 +9,32 @@ use anyhow::{anyhow, Context, Result};
 use dialoguer::{theme::ColorfulTheme, Confirm};
 use ilias::{client::IliasClient, exercise::grades::GradePage, local_file::NamedLocalFile};
 use log::{debug, info};
+use once_cell::sync::Lazy;
 use regex::Regex;
 
+static FLATTENED_FILENAME_REGEX: Lazy<Regex> = Lazy::new(|| {
+    Regex::new("@student\\.kit\\.edu_\\d+-(?<filename>.*)").expect("Invalid regex pattern")
+});
+
```

Comment:
While I don't really see extracting this as necessary here as `upload_feedback` is only called once, this general pattern is very common in ilias-rs. As I'm not very happy with my current implementation there I'm planning to move to https://crates.io/crates/lazy_static, so I would suggest using this here as well (or not extracting the regex into a static variable)